### ISSUE TITLE
Two identical posts are generated at once with GF 2.5.x

### ISSUE THREAD

**Issue by lhberg** [pods-framework/pods-gravity-forms#154]
## Issue Overview

Two identical posts are generated at once
## Expected Behavior

only one post should be generated
## Current Behavior

Two identical posts are generated at once
## Possible Solution

Most probably an issue with GF 2.5.x
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Form Upload

<details>
```
Upload Gravity Form JSON or Copy and Paste it Here.
```
</details>

## Pods Package Export

<details>
```
Copy and Paste the JSON Export from Pods Migrate: Packages, Export and Paste it Here
```
</details>

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

**Comment by sugarfunk** [comment]
Also confirmed GF 2.5.1

**Comment by AurelienEC** [comment]
I confirm what mentioned @HmCody with the option support entry update it stop duplicate the entry.

**Comment by FormlosLK** [comment]
> I confirm what mentioned @HmCody with the option support entry update it stop duplicate the entry.

Can confirm this too.

**Comment by Brian-Milnes** [comment]
I'll test the "Support entry updates" but this version (1.4.4) is also failing to attach PDF uploads (which reverting to 1.4.3 resolves)

**Comment by sc0ttkclark** [comment]
I'm in the midst of a major refactor for Pods Gravity Forms 2.0 which will include a full retest of all supported functionality. This will happen after Pods 2.9 is released since one of the Pods GF 2.0 features will be to support GF list fields mapping into simple repeatable fields.

Thanks for your continued patience, I'll keep looking at this to see if I can potentially solve this before then.

**Comment by danielcharrua** [comment]
Any solutions for this matter?

**Comment by Brian-Milnes** [comment]
I'd like to know too, as I cannot update to the latest version because of this issue.
(not good to miss out on other bug-fixes and enhancements)

**Comment by jmbullis** [comment]
Reporting that the issue is still happening. I used the Suppor Entry Updates workaround and it works for now.

**Comment by Brian-Milnes** [comment]
Thanks for the reminder, James
I am still stuck on 1.4.3, but need latest to get functionality.
@sc0ttkclark can you take a look, please?

**Comment by sclark3-godaddy** [comment]
I am 'off work' starting next week until new years day, I'm planning on putting more time into the Pods GF 2.0 release and this is on my list. Thanks for chiming in as often times it's going to be the issues people have recently responded to that will get attention first :)

**Comment by sc0ttkclark** [comment]
I am 'off work' starting next week until new years day, I'm planning on putting more time into the Pods GF 2.0 release and this is on my list. Thanks for chiming in as often times it's going to be the issues people have recently responded to that will get attention first :)

**Comment by Brian-Milnes** [comment]
Hi Scott
Once you have gotten over the new Pods release, could you please take another look at this, please?
There are the two issues.

1. The creation of a duplicate entry on submission of the form 
2. Failure to attach a PDF on submission

I am looking to embark on a Gravity Flow implementation and this relies on this...
Thanks
Brian

**Comment by sc0ttkclark** [comment]
I'm not seeing the duplicate entry handling but that may have been resolved in Pods GF 1.4.6 already. Perhaps some of you can verify with that release once it's out.

**Comment by sc0ttkclark** [comment]
Fixed in Pods GF 1.5, confirmed with another site having that issue

**Comment by sc0ttkclark** [comment]
Pods GF 1.5 was released today
